helm/opendistro-es/values.yaml

```diff
@@ -557,3 +557,38 @@ curator:
       loglevel: INFO
       logformat: default
       blacklist: ['elasticsearch', 'urllib3']
+
+configurer:
+  enabled: false
+
+  # If true, configurer will create an index for each template if there is
+  # none.
+  createIndices: true
+
+  # If true, configurer will update/override an index policy if it already
+  # exists.
+  updatePolicies: true
+
+  # If true, configurer will only create new templates, not update.
+  # See 'create' parameter:
+  # https://opendistro.github.io/for-elasticsearch-docs/docs/elasticsearch/rest-api-reference/#indicesput_template
+  onlyCreateTemplates: false
```

Comment:
```suggestion
  # If true, configurer will overwrite existing templates.
  # See 'create' parameter:
  # https://opendistro.github.io/for-elasticsearch-docs/docs/elasticsearch/rest-api-reference/#indicesput_template
  overwriteTemplates: true
```

Reply:
This would be a negation of the API, which is why I implemented it as I did. So, pretty much to keep configuration similar to opendistro for those familiar to it.
I agree that `overwriteTemplates` looks better though.
What do you think?

Reply:
Yeah, now that i read it in the documention it sort of sounds overly complicated.
If i get to choose I would go with the negation and keep `overwriteTemplates: true` :slightly_smiling_face:

Reply:
It's fine to revert if you want if, if the previous solution is cleaner. This is totaly fine with me :slightly_smiling_face:

Reply:
Both are fine I guess, can keep it unless anyone objects :slightly_smiling_face: